swarm/storage/dpa_test.go

```diff
@@ -86,13 +86,15 @@ func TestDPArandom(t *testing.T) {
 }
 
 func TestDPA_capacity(t *testing.T) {
-	tdb, err := newTestDbStore(false)
+	tdb, err := newTestDbStore(false, false)
 	if err != nil {
 		t.Fatalf("init dbStore failed: %v", err)
 	}
 	defer tdb.close()
 	db := tdb.LDBStore
-	memStore := NewMemStore(db, 0)
+	storeParams := NewStoreParams(0, nil, nil)
```

Comment:
It is very difficult to reason about what `NewStoreParams` is doing here. I think we should revise this constructor, and others that are built in this way - concretely avoid using `nil` in constructors, as well as booleans.

Reply:
Ah, I see I am not the only one :)